api/models/process.go

```diff
@@ -247,32 +249,32 @@ func ListProcesses(app string) ([]*Process, error) {
 		select {
 		case ps := <-psch:
 			pss = append(pss, &ps)
-		case err := <-errch:
-			return nil, err
 		}
 	}
 
 	pending, err := ListPendingProcesses(app)
-
 	if err != nil {
-		return nil, err
+		fmt.Printf("ns=kernel at=ListProcesses state=error message=\"unable to get pending processes: %s\"\n", err)
```

Comment:
No direct error was observed. It was strictly "failing gracefully" in the event that an ECS call doesn't work out.